app/config/FeatureSwitches.scala

```diff
@@ -18,24 +18,32 @@ package config
 
 import play.api.Configuration
 
-case class FeatureSwitch(value: Option[Configuration]) {
+case class FeatureSwitches(featureSwitchConfig: Configuration) {
 
   private val versionRegex = """(\d)\.\d""".r
 
   def isVersionEnabled(version: String): Boolean = {
-    val versionNoIfPresent: Option[String] =
+    val maybeVersion: Option[String] =
       version match {
         case versionRegex(v) => Some(v)
         case _               => None
       }
 
     val enabled = for {
-      versionNo <- versionNoIfPresent
-      config    <- value
-      enabled   <- config.getOptional[Boolean](s"version-$versionNo.enabled")
+      validVersion <- maybeVersion
+      enabled      <- featureSwitchConfig.getOptional[Boolean](s"version-$validVersion.enabled")
     } yield enabled
 
     enabled.getOrElse(false)
   }
 
+  val isV1R7cRoutingEnabled: Boolean        = isEnabled("v1r7c-endpoints.enabled")
```

Comment:
Why do we need `isV1R7cRoutingEnabled` and `isTaxYearNotEndedRuleEnabled` here? They weren't in the code before so wondering what the reason is for needing them?